src/Resolver/AccessResolver.php

```diff
@@ -29,24 +28,27 @@ public function __construct(PromiseAdapter $promiseAdapter)
     /**
      * @return Promise|mixed|Connection
      */
-    public function resolve(callable $accessChecker, callable $resolveCallback, array $resolveArgs = [], bool $useStrictAccess = false)
+    public function resolve(callable $accessChecker, callable $resolveCallback, array $resolveArgs = [], array $accessConfig = [])
     {
+        $useStrictAccess = $accessConfig['useStrictAccess'];
```

Comment:
please inject variables, array make it more difficult to debug since we loose exact  variables list or type here

Reply:
Done